mail_debrand/models/mail_template.py

```diff
@@ -49,3 +54,17 @@ def _debrand_body(self, html):
     def render_post_process(self, html):
         html = super().render_post_process(html)
         return self._debrand_body(html)
+
+    def _get_template_lang(self):
+        template_lang = False
+        if self._context.get("default_template_id"):
+            template = self.env["mail.template"].browse(
+                self._context.get("default_template_id")
+            )
+            if template.lang and self._context.get("active_id"):
+                langs = template._render_template(
+                    template.lang, template.model, [self._context.get("active_id")]
+                )
+                for _res_ids, lang in langs.items():
+                    template_lang = lang
```

Comment:
The field lang, model mail.template, should have something like this: ${object.partner_id.lang}

`template_lang = template._render_template(
                    template.lang, template.model, self._context.get("active_id")
                )`

I applied an adjustment, because it is true that only one valid value will be obtained from that mako expression